site/content/blog/2024-11-10-powering-up-dast-with-zap-and-noir/index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Powering Up DAST with ZAP and Noir"
+date: "2024-11-10"
+summary: "Integrating Noir, a tool for discovering hidden endpoints in source code, with ZAP enhances dynamic application security testing (DAST)."
+type: post
+tags:
+  - blog
+  - automation
+  - recon
+authors:
+  - hahwul
+---
```

Comment:
I'd suggest setting the title image for the social preview.

Reply:
The syntax is:
```
images:
- https://www.zaproxy.org/blog/2023-08-25-community-tips-and-tricks/images/zapbot.TandT.gif
addSocialPreview: true
```

Reply:
The `addSocialPreview` is true by default for blog posts.